py_bind/test/test_MultimediaParams.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from optv.parameters import MultimediaParams
+
+class Test_MultimediaParams(unittest.TestCase):
+    
+    @staticmethod
+    def identical_lists(self, list_1, list_2):
```

Comment:
You can compare lists directly in Python, and the comparison operator basically implements your logic. Note this IPython session:
```
In [1]: [1, 2, 3] == [1, 2, 3]
Out[1]: True

In [2]: [1, 2, 3] == [1, 2, 3.]
Out[2]: True

```